support/index.html

```diff
@@ -81,6 +81,16 @@ <h4>OME Training</h4>
                     </div>
                 </div>
             </div>
+            <div class="small-12 medium-4 columns">
+                <div class="card card-support">
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-link fa-2x"></i>
+                        <h4>On the web</h4>
+                        <p class="card-caption">Follow us on social media or listen in to a community meeting. Zoom link on request via social media or <a href="https://forum.image.sc/g/ome">image.sc</a></p>
```

Comment:
```suggestion
                        <p class="card-caption">Follow us on social media, read minutes of or listen in to an OME Team meeting. Zoom link on request via social media or <a href="https://forum.image.sc/g/ome">image.sc</a></p>
```

Reply:
Deleting this suggestion, as it makes the text in the box too long and thus misformats the layout. Will push a commit from local instead.